.structure-config

```diff
@@ -0,0 +1 @@
+INFRASTRUCTURE_FILES='.structure-config .packit.yml .pep8speaks.yml .codecov.yml .coveragerc .github/ dockerfile/ scripts/testing/ scripts/githooks/'
```

Comment:
`.coveragerc` is part of anaconda-33.16.5.2-1.el8.src.rpm - is it expected? If yes, then is should not be marked as an infrastructure file.

Reply:
I don't think we want to have `.coveragerc` in the tarball but I could be wrong. @poncovka do you know why it is in tarball?

Reply:
For historical reasons (https://github.com/rhinstaller/anaconda/commit/b9dcc94b5e114e9bd03b441342ccfa131d7a9298)? I think we can drop it.

Reply:
OK, I'll add new commit to drop the `.coveragerc` from tarball. Thanks!